scripts/apps/search/components/MarkedDesksList.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {
+    closeActionsMenu
+} from 'apps/search/helpers';
+
+export class MarkedDesksList extends React.Component {
```

Comment:
Isn't it the case for all the other jsx files you commited yesterday or am I missing something?